scenarios/scenario-10/README.md

```diff
@@ -0,0 +1,116 @@
+## Scenario 10 - Apply interceptors to the API
+- This scenario describes how to apply interceptors to carry out transformations and mediations on the requests and responses.
+- First, we need to implement custom request interceptors and response interceptors. For more information about implementing custom interceptors you can refer the document [adding interceptors.](https://docs.wso2.com/display/MG300/Adding+Interceptors)
+- Then we need to Initialize a new WSO2 API Manager project and add the custom interceptor files in interceptors folder.
```

Comment:
we need to initialize API project. (WSO2 API Manager project -> API project)

Reply:
done